### PR TITLE
Issue #12145: corrected tokens so all are required

### PR DESCRIPTION
**Pull request by rnveach**
Issue #12145

it is breaking compatibility .... will need review from all.

### CODE REVIEW

**Review by strkkk — APPROVED** [review]

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder_HonorsTokensProperty.java [hunk 1]
```diff
@@ -17,8 +17,8 @@
 package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
 
 import static javax.swing.WindowConstants.DISPOSE_ON_CLOSE; // ok
-import static java.awt.Button.ABORT; // ok
-import java.awt.Dialog; // ok
+import static java.awt.Button.ABORT; // violation
+import java.awt.Dialog; // violation
```
Comment (nrmancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder_HonorsTokensProperty.java:21]: Please add violation message to comment.

Comment (nrmancuso) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder_HonorsTokensProperty.java:21]: Also, we need to update config for this file.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Items:

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder_HonorsTokensProperty.java (hunk 1 above)
Comment (rnveach) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder_HonorsTokensProperty.java:21]: Please clarify how you want config file updated. I left it alone to show this is how the check will function for users who did use this weird configuration before this change.

Do you wish me to remove the tokens completely?

**Review by pbludov — APPROVED** [review]

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder_HonorsTokensProperty.java (hunk 1 above)
Comment (nrmancuso) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder_HonorsTokensProperty.java:21]: Let's just remove `tokens`

Comment (rnveach) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder_HonorsTokensProperty.java]: Done both.

Removed tokens from config header and added violation message to comment.

**Review by nrmancuso — APPROVED** [review]

**Comment by rnveach** [comment]
@romani ping

**Review by romani — APPROVED** [review]
ok to merge

**Pull request merged**